ksql-engine/src/test/resources/query-validation-tests/geodistance.json

```diff
@@ -0,0 +1,35 @@
+{
+  "tests": [
+    {
+      "name": "geo_distance",
+      "statements": [
+        "CREATE STREAM TEST (ID bigint, LAT1 double, LON1, double, LAT2, double, LON2 double, RADIUS varchar) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE STREAM DISTANCE_STREAM AS select ID, geo_distance(LAT1, LON1, LAT2, LON2, RADIUS) as CALCULATED_DISTANCE from test;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "value": {"ID": 1, "LAT1": 37.4439, "LON1": -122.1663, "LAT2": 51.5257, "LON2": -0.1122}},
+        {"topic": "test_topic", "value": {"ID": 2, "LAT1": 37.4439, "LON1": -122.1663, "LAT2": 51.5257, "LON2": -0.1122}, "RADIUS": "KM"},
+        {"topic": "test_topic", "value": {"ID": 3, "LAT1": 37.4439, "LON1": -122.1663, "LAT2": 51.5257, "LON2": -0.1122}, "RADIUS": "MI"},
+        {"topic": "test_topic", "value": {"ID": 4, "LAT1": 51.5257, "LON1": -0.1122, "LAT2": -33.9323, "LON2": 18.4197}},
```

Comment:
nit: this is a good test to have, but I meant to have another test suite with
```
      "statements": [
        "CREATE STREAM TEST (ID bigint, LAT1 double, LON1, double, LAT2, double, LON2 double) WITH (kafka_topic='test_topic', value_format='JSON');",
        "CREATE STREAM DISTANCE_STREAM AS select ID, geo_distance(LAT1, LON1, LAT2, LON2) as CALCULATED_DISTANCE from test;"
      ],
```
To test the case where the UDF is called with only 4 params (not a null 5th param)